routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: admin.orders.web.ch.gov.uk
+group: web
+weight: 1000
+routes:
+  1: ^/ordersAdmin/.*
```

Comment:
Please use hyphen case for path segments; e.g:

```yaml
routes:
    1: ^/orders-admin/.*
```

Reply:
Resolved in latest commit